### PR TITLE
fix(composer): Fix camera view positioning under sub model

### PR DESCRIPTION
**Pull request by jwills-jdubs**
## Overview
Camera's do not get the right positioning when placed on a SubModelRef. This fix uses the non-submodel space for a sub model while maintaining hierarchical placement

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx [hunk 1]
```diff
@@ -209,13 +209,21 @@ export const AddObjectMenu = () => {
         parentRef: getRefForParenting(),
       } as unknown as ISceneNodeInternal;
 
-      const parent = getObject3DBySceneNodeRef(node.parentRef);
+      let physicalParentNode: ISceneNodeInternal | undefined = getSceneNodeByRef(node.parentRef);
+      console.log('start', physicalParentNode);
```
Comment (mumanity) [inline packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx:213]: Were these intentionally left behind?

**Review by mumanity — APPROVED** [review]

File: packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx (hunk 1 above)
Comment (jwills-jdubs) [reply packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx:213]: Fudge

File: packages/scene-composer/src/utils/nodeUtils.ts [hunk 2]
```diff
@@ -124,12 +124,15 @@ export const createNodeWithTransform = (
   rotation: THREE.Euler,
   scale: THREE.Vector3,
   parent?: THREE.Object3D,
+  targetRef?: string,
 ): ISceneNodeInternal => {
   const finalTransform = getFinalTransform({ position, rotation, scale }, parent);
 
   const node = (newWidget as AddingWidgetInfo).node ? (newWidget as AddingWidgetInfo).node : newWidget;
+  console.log('nodeInfo', node);
```
Comment (mumanity) [inline packages/scene-composer/src/utils/nodeUtils.ts]: One more...

**Review by mumanity — APPROVED** [review]

File: packages/scene-composer/src/utils/nodeUtils.ts (hunk 2 above)
Comment (jwills-jdubs) [reply packages/scene-composer/src/utils/nodeUtils.ts]: Double Fudge brownie

**Pull request merged**